README.md

```diff
@@ -22,13 +24,16 @@ docker run \
     --mount "type=bind,src=YOUR_OUTPUT,dst=/output_dir" \
     ome-tiff-tiler
 ```
+
 (Or `gehlenborglab/ome-tiff-tiler:latest` to pull from DockerHub.)
 
-Two environment variables should be set:
-- `CHANNEL_PAGE_PAIRS`: Space-delimited pairs of layer name and corresponding TIFF index.
-- `PREFIX`: Prefix to prepend to tile filenames.
+Four environment variables should be set:
 
-And two mounts should be provided:
+- `CHANNEL_PAGE_PAIRS`: (Only for use with `dz` pyramid type) Space-delimited pairs of layer name and corresponding TIFF index.
+- `PREFIX`: Prefix to prepend to tile filenames.
+- `SERVER_URL`: (Only for use with `tiff` pyramid type) For creating metadata that can be used as a JSON, put in the root server_url
```

Comment:
This is unclear to me?

Reply:
If you're serving from [example.com](example.com), an ome tif will be stored at [example.com/vanderbilt_MxIF.images/xyz.ome.tif](example.com/vanderbilt_MxIF.images/xyz.ome.tif) and I am trying to capture that